nomad/structs/connect.go

```diff
@@ -6,6 +6,7 @@ package structs
 // ConsulConfigEntries represents Consul ConfigEntry definitions from a job for
 // a single Consul namespace.
 type ConsulConfigEntries struct {
+	Cluster     string
 	Ingress     map[string]*ConsulIngressConfigEntry
 	Terminating map[string]*ConsulTerminatingConfigEntry
 }
```

Comment:
I may be missing something about Consul's config entry pattern, but this struct seems a bit odd to me now 🤔  

The godoc mentions that a `ConsulConfigEntries` is for a a single Consul namespace, and I assume a Consul namespace is specific to a cluster?

If that's the case, then I think I would expect `job.ConfigEntries` below to return a `map[string]map[string]*ConsulConfigEntries` with the `cluster -> namespace -> CE` mapping? But that's also a bit odd to me, so adding both `Namespace` and `Cluster` would make things more clear?

Reply:
A Consul namespace is specific to a Consul cluster, but not necessarily unique across clusters. But we don't let you set a Consul namespace on the `service`, only on the group.